apps/pie-storybook/stories/pie-textarea.stories.ts

```diff
@@ -42,6 +95,41 @@ const textareaStoryMeta: TextareaStoryMeta = {
                 summary: defaultProps.resize,
             },
         },
+        name: {
+            description: 'The name of the input (used as a key/value pair with `value`). This is required in order to work properly with forms.',
+            control: 'text',
+            defaultValue: {
+                summary: defaultArgs.name,
+            },
+        },
+        value: {
+            description: 'The value of the textarea (used as a key/value pair in HTML forms with `name`).',
+            control: 'text',
+            defaultValue: {
+                summary: defaultProps.value,
+            },
+        },
+        readonly: {
+            description: 'When true, the user cannot edit the control. Not the same as disabled. See [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/readonly) for more information.',
+            control: 'boolean',
+            defaultValue: {
+                summary: false,
+            },
+        },
+        defaultValue: {
+            description: 'An optional default value to use when the textarea is reset.',
+            control: 'text',
+            defaultValue: {
+                summary: '',
+            },
+        },
+        required: {
```

Comment:
I noticed that `autocomplete` and `autoFocus` are missing storybook controls. Was this intentional? I think autoFocus might be useful to have, and perhaps autocomplete inside the form story